content/java/concepts/strings/terms/trim/trim.md

```diff
@@ -0,0 +1,47 @@
+---
+Title: '.trim()'
+Description: 'Removes leading and trailing whitespace from a string.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'String'
+  - 'Methods'
+  - 'Characters'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.trim()`** method removes any leading and trailing whitespace from a string.
+
+## Syntax
+
+```pseudo
+string.trim()
+```
+
+No parameters are required for the `.trim()` method.
+
+## Example
+
+The `.trim()` method is showcased in the following example:
+
+```java
+// Example.java
+import java.io.*;
+
+class Example {
+  public static void main(String[] args) {
+    String greetings = "   Hello, Stranger!  ";
+    System.out.println(greetings.trim());
+  }
+}
+```
+
+In the example below, the `.trim()` method removes leading and trailing whitespaces from the string `greetings`.
```

Comment:
@MiiPoikaj I think my previous note may have been confusing. I was suggesting that you replace line 27 with 41. Having both is unnecessary, and 41 is the more descriptive of the two. Let me know if you have any additional questions.